### PR TITLE
Prepare for 0.6.0 release

### PR DESCRIPTION
**Pull request by reyang**

### CODE REVIEW

File: contrib/opencensus-ext-grpc/version.py [hunk 1]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.3.dev0'
+__version__ = '0.3'
```
Comment (songy23) [inline contrib/opencensus-ext-grpc/version.py]: `0.3.0`?

File: context/opencensus-context/version.py [hunk 2]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.dev0'
+__version__ = '0.1.0'
```
Comment (songy23) [inline context/opencensus-context/version.py]: Should this be `0.2.0` or `0.1.1`?

File: contrib/opencensus-correlation/version.py [hunk 3]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.dev0'
+__version__ = '0.1.0'
```
Comment (songy23) [inline contrib/opencensus-correlation/version.py]: Same here.

File: context/opencensus-context/version.py (hunk 2 above)
Comment (reyang) [reply context/opencensus-context/version.py]: Current version is 0.1.0, and there is no change, so we don't need a new release.

File: contrib/opencensus-ext-grpc/version.py (hunk 1 above)
Comment (reyang) [reply contrib/opencensus-ext-grpc/version.py]: You're right, fixed. Thanks!

File: contrib/opencensus-correlation/version.py (hunk 3 above)
Comment (reyang) [reply contrib/opencensus-correlation/version.py]: There is no change, probably no need to release this package.

Comment (reyang) [reply contrib/opencensus-correlation/version.py]: Just noticed this one is special, the version should be 0.3.0 although it hasn't been changed at all.

Comment (reyang) [reply contrib/opencensus-correlation/version.py]: Changed to 0.3.0.

**Pull request merged**